django/template/base.py

```diff
@@ -122,6 +122,9 @@ def __init__(self, name, template_name=None, loader=None):
     def __str__(self):
         return self.name
 
+    def __repr__(self):
+        return f'{self.__class__.__qualname__}(name={self.name})'
```

Comment:
Typically repr outputs include quoting around strings. One of the original ideas behind repr was that you should be able to execute the result and get a valid Python object back.

```suggestion
        return f'{self.__class__.__qualname__}(name=“{self.name}”)'
```

Reply:
A better way to do this is with `{self.name!r}`.

Reply:
Hi @orf and @cjerdonek, completely forgot the quotes :man_facepalming:, thanks a lot for the feedback here!